client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1523,12 +1524,11 @@ Undocumented.prototype.fetchDns = function( domainName, fn ) {
 	this.wpcom.req.get( '/domains/' + domainName + '/dns', fn );
 };
 
-Undocumented.prototype.addDns = function( domain, record, fn ) {
```

Comment:
What about the code in https://github.com/Automattic/wp-calypso/blob/51909254a28043c4ab93a5fc0932353825597c37/client/my-sites/upgrades/domain-management/dns/dns-list.jsx#L11 and https://github.com/Automattic/wp-calypso/blob/51909254a28043c4ab93a5fc0932353825597c37/client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx#L93 that still uses this action? :)